components/eam/src/control/cam_history.F90

```diff
@@ -1761,6 +1774,8 @@ subroutine read_restart_history (File)
         ierr = pio_get_var(File,zonal_complement_desc, (/f,t/), tape(t)%hlist(f)%field%zonal_complement)
         ierr = pio_get_var(File,avgflag_desc, (/f,t/), tape(t)%hlist(f)%avgflag)
         ierr = pio_get_var(File,longname_desc, (/1,f,t/), tape(t)%hlist(f)%field%long_name)
+        tape(t)%hlist(f)%field%standard_name = ''
```

Comment:
seems weird to mix this initialization line in with a block of pio_get_var lines. I think it'd be cleaner to move it down below.

Reply:
I am not sure if it can be moved down, the default inits later on in the code is for attributes that are not saved (so not read back)
@dqwu: Can you remove the above initialization? Currently long_name and other attributes don't have any initialization (if you want to add these inits maybe add them in a block above all the get_vars, but add it for all the field attributes)

Reply:
Now looking at the code below, sampling_seq field has an initialization just before its read back. So I am ok with the init here as is. I will approve the PR soon.